src/Block/AdminStatsBlockService.php

```diff
@@ -33,7 +33,7 @@ class AdminStatsBlockService extends AbstractBlockService
     /**
      * @param string $name
```

Comment:
you can remove this and add `string` typehint to name

Reply:
Sure. Then i will do it for `AbstractBlockService::__construct` in BlockBundle first